src/data/lilycove_lady.h

```diff
@@ -2,13 +2,52 @@
 #include "constants/items.h"
 #include "constants/moves.h"
 
-static const u16 sContestLadyMonGfxId[] =
-{
-    [CONTEST_CATEGORY_COOL]   = OBJ_EVENT_GFX_ZIGZAGOON_1,
-    [CONTEST_CATEGORY_BEAUTY] = OBJ_EVENT_GFX_SKITTY,
-    [CONTEST_CATEGORY_CUTE]   = OBJ_EVENT_GFX_POOCHYENA,
-    [CONTEST_CATEGORY_SMART]  = OBJ_EVENT_GFX_KECLEON,
-    [CONTEST_CATEGORY_TOUGH]  = OBJ_EVENT_GFX_PIKACHU
+struct LilycoveContestLadyValues
+{
+    u16 monGfxId;
+    u16 monSpecies;
+    const u8 *monName;
+    const u8 *categoryName;
+    const u8 *contestName;
+};
+
+static const struct LilycoveContestLadyValues sContestLadyValues[] =
+{
+    [CONTEST_CATEGORY_COOL] = {
+        .monGfxId = OBJ_EVENT_GFX_ZIGZAGOON_1,
+        .monSpecies = SPECIES_ZIGZAGOON,
+        .monName = gText_ContestLady_Handsome,
+        .categoryName = gText_ContestLady_Coolness,
+        .contestName = gText_CoolnessContest,
```

Comment:
These are only used once, so I would personally be inclined to convert them to ``COMPOUND_STRING``s and remove them from ``strings.c`` and ``strings.h``. The only reason they're lumped with the rest of the strings is for pret matching purposes and it just makes it harder to edit imo

Reply:
I went one step further and converted the other Lilycove Lady strings into COMPOUND_STRINGs too